### ISSUE TITLE
Implement Filter on stories by description.

### ISSUE THREAD

**Issue by Swastikyadav** [Swastikyadav/hacker-news-clone#6]
- Every story either have the key `text` or not.
- This `text` key represents the description for that story.
- Use it to filter stories by description.

## Scope
- Filter on the basis of whether a story have description or not.

![image](https://user-images.githubusercontent.com/44374494/192534107-941d2db7-78c5-46a2-8d30-16dd0b908744.png)

**Comment by khalatevarun** [comment]
Could you please elaborate on this issue a little more?
Does filter mean we don't have to show news which don't have description ?

**Comment by ChaudharyRaman** [comment]
Hi @Swastikyadav, I would like to work on this issue . Can you please assign it to me?

**Comment by Swastikyadav** [comment]
> Could you please elaborate on this issue a little more? Does filter mean we don't have to show news which don't have description ?

Yes, when filtered by description it should only show stories with description, and when filter is removed, it should again show all the stories.
